app/services/state_file/state_information_service.rb

```diff
@@ -27,11 +27,19 @@ def active_state_codes
       def state_intake_classes
         STATES_INFO.map { |_, attrs| attrs[:intake_class] }
       end
-      
+
       def state_intake_class_names
         state_intake_classes.map(&:to_s).freeze
       end
 
+      def state_name(state_code)
+        unless active_state_codes.include?(state_code)
+          raise StandardError, state_code
+        end
+
+        STATES_INFO[state_code][:state_name]
+      end
+
```

Comment:
so this one is already defined above on lines 4-20! anything that's just a getter for one of the attributes should go in that list

Reply:
Good catch - this was the result of a merge.